markdown/blog/2024-mentorship-metrics.md

```diff
@@ -0,0 +1,94 @@
+    byline: AsyncAPI Preacher
+excerpt: "AsyncAPI Mentorship Program: A Deep Dive into the Stats"
+---
+
+The AsyncAPI mentorship program is a global online initiative designed to help new and existing contributors make significant contributions to AsyncAPI projects. The program pairs participants with seasoned mentors and maintainers, offering a structured path for growth over a period of time.
+
+<Figure
+  src="/img/posts/mentorship-assets/overview.webp"
+  caption="Overview of 2024 Mentorship Program metrics"
+  className="text-center"
+/>
+
+## Contributors
+
+- 65.04% of selected contributors had little to no experience contributing to open source.
+- 80% of contributors were active communicators throughout the program. 
+- 77.78% of selected contributors followed best practices while contributing. 
+
+## Projects
+
+- 37.5% of accepted project proposals were aimed at project maintainership.
+- 90% - a project completion rate.
+- 85.7% of contributors agreed that mentors provided adequate guidance and support to achieve their project goals.
+
+## Mentors
+
+<Figure
+  src="/img/posts/mentorship-assets/pie.webp"
+  caption="Mentor composition and evaluation outcomes"
+  className="text-center"
+/>
+
+The 2024 program welcomed eight highly esteemed and experienced mentors, 80% of whom are active maintainers.
+
+- Two-thirds of accepted mentors are returning mentors from previous editions of the program.
+- 85.7% of mentors passed the mentorship evaluations. 
+- 77.78% of mentors provided mentees with an exceptional mentorship experience.
+
+## Program
+
+<Figure
+  src="/img/posts/mentorship-assets/line-graph.webp"
+  caption="Reported learning impact across the program timeline"
+  className="text-center"
+/>
+
+90% of contributors reported that the program made a meaningful contribution to their learning, helping them gain valuable skills, knowledge, and hands-on experience that supported their growth throughout the program.
+
+### Feedback
+
+> Thank you for giving me to opportunity, and that my background does not define me contributing to OSS.
+
+> The program is already doing well. I believe if the suggestions made by Ace during one of our community meetings to add certifications are included then the program will be even better.
+
+> The program is just amazing for contributors like me who are starting their journey.
+
+> First off, I'll say a very big Thank you!
+
+> I've grown so much in this program, and I owe it
+
+> Please don't take initiatives that are out of your bandwith, like to get some feedback of my work I had to personally dm 5 to 6 people and except derberg and ace sometimes the rest don't give a damn.
+
+> The program goal is clear. But one thing that lukasz always say that we have to focus more on maintainership. Including projects that require maintainership will be very helpful to the community. I agree to this point, it will be giving AsyncAPI community a lot of dedicated contributors.
```

Comment:
```suggestion
> The program goal is clear. But one thing that Lukaz always say that we have to focus more on maintainership. Including projects that require maintainership will be very helpful to the community. I agree to this point, it will be giving AsyncAPI community a lot of dedicated contributors.
```